libs/execution_context/ebpf_core.c

```diff
@@ -139,6 +141,9 @@ static const void* _ebpf_general_helpers[] = {
     (void*)&_ebpf_core_memmove,
     // No default implementation of bpf_get_socket_cookie
     (void*)NULL, // bpf_get_socket_cookie
+    (void*)&_ebpf_core_strcpy,
+    (void*)&_ebpf_core_strcat,
```

Comment:
why strcpy and strcat here instead of strncpy_s and strncat_s?

Reply:
No good reason; I'll make those match.